lib/cs_guide/resources/venue.ex

```diff
@@ -9,11 +9,19 @@ defmodule CsGuide.Resources.Venue do
 
   schema "venues" do
     field(:venue_name, :string)
-    field(:postcode, :string)
-    field(:phone_number, :string)
+    field(:postcode, Fields.Postcode)
+    field(:phone_number, Fields.PhoneNumber)
     field(:cs_score, :float, default: 0.0)
+    field(:num_cocktails, :integer)
     field(:entry_id, :string)
     field(:deleted, :boolean, default: false)
+    field(:description, :string)
+    field(:website, :string)
+    field(:address_line_1, Fields.Address)
+    field(:city, Fields.Address)
```

Comment:
@Danwhy this should be `field(:city, Fields.city)` 
the Ecto type should be limited to 85 chars as per:
https://www.worldatlas.com/articles/the-10-longest-place-names-in-the-world.html
and the UI should be an autocomplete. 

We can _merge_ this "as is" for "demo fix" purposes, 
but we need to figure out how to make the _best_ possible UX in a future iteration.